.vscode/settings.json

```diff
@@ -2,5 +2,7 @@
   "files.associations": {
     "*.overlay": "dts",
     "*.keymap": "dts"
-  }
+  },
+  "C_Cpp.dimInactiveRegions": false,
```

Comment:
We need to generally solve the "how to manage settings.json", but for now, can you please revert these changes? Thanks.